templates/ep19/bs/conference/talk_social_card.html

```diff
@@ -64,7 +64,7 @@
   <h2>{{ subtitle }}</h2>
   <h1>{{ title }}</h1>
 
-  <img src="{% static "img/ep2020-logo-social-card.png" %}" />
```

Comment:
I wouldn't change it unless you've confirmed this page looks as expected - it uses the logo with a transparent background to generate a per-talk card like this:

![image](https://user-images.githubusercontent.com/3726919/75628503-f2566300-5bd9-11ea-80bf-7ab6aee00aaf.png)

Reply:
Where is the logic for this in the code ? I wasn't aware that the website combines the talk title with the logo. The per talk card looks ok, but the color is wrong for EP2020 and it's probably better to change the talk card back to use the monochrome version of the logo or to integrate it with the background image we're using. In any case, the talk card is still too high for social media. I'll open a separate ticket for this.

Reply:
> The per talk card looks ok, but the color is wrong for EP2020 

The link and the logo above is for ep2019. The logic for this is in the view that renders the `talk_social_card.html` template, I don't remember off the top of my head.